Created badges documentation (#260)
mozilla-l10n/localizer-documentation: src/tools/pontoon/notifications.md

This notification is sent to team managers, as soon as a new user makes the first contribution to their team.

### Badge awarded

> This notification is sent to a user upon being awarded with a [badge or badge level](profile.md#achievement-badges) from reviewing and submitting translations, or promoting users.

Reviewer: ```suggestion
This notification is sent to a user upon being awarded with a [badge or new badge level](profile.md#achievement-badges) from reviewing and submitting translations, or promoting users.
```

Maybe this?

> This notification is sent to a user upon gaining a new [badge level](profile.md#achievement-badges) from reviewing a translation, submitting a translation, or promoting a user.

Reviewer: ```suggestion
This notification is sent upon gaining a new [badge level](profile.md#achievement-badges) from reviewing and submitting translations, or promoting users.
```

This notification is sent to team managers, as soon as a new user makes the first contribution to their team.

### Badge awarded

> This notification is sent to a user upon being awarded with a [badge or badge level](profile.md#achievement-badges) from reviewing and submitting translations, or promoting users.

Reviewer 2: One isn't really awarded a badge level. Can we make that clearer somehow?

```suggestion
This notification is sent to a user upon being awarded with a [badge or badge level upgrade](profile.md#achievement-badges) from reviewing and submitting translations, or promoting users.
```
